CollAction/Models/ProjectViewModels/ParticipatingInProjectsViewModel.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.Extensions.Localization;
+
+namespace CollAction.Models.ProjectViewModels
+{
+    public class ParticipatingInProjectsViewModel : FindProjectsViewModel
+    {
+        public ParticipatingInProjectsViewModel(IStringLocalizer localizer) : base(localizer)
```

Comment:
We need to either start localizing, or stop using the localizer. It's used everywhere, but we're not localizing anything. Something for the future.

Reply:
True.. case of YAGNI I suppose. I'll leave it for now. Localisation will be a challenge when trying to tie into React though. Mvc.net and React don't really work together in that respect.